modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -306,25 +404,27 @@ To archive this goal some Breadcrumbs API has been reviewed and the next changes
 </Breadcrumbs>
 ```
 
-🙎‍♂️ The next changes will require manual changes:
+Additionally, there are a handful of changes to the `Breadcrumbs` API which will require manual
+updates on your part:
 
-- Handling Redirects
+- Changes to Redirects
 - Updated Overflow Behavior
 
-#### Handling Redirects
+We've outlined these changes in more detail below.
 
-`onAction` has been removed from `Breadcrumbs.Item`, so the way of redirection has been changed.
+#### Changes to Redirects
 
-`Breadcrumbs.Link` defaults to redirecting with an `href`, which means the page will hard-redirect
-when the link is clicked. However, if you're in a single-page application (SPA) environment, you
-might want to use the internal SPA router instead. You can override that behavior with a custom
-`onClick` handler that blocks the default behavior of the event and passes along the link path to
-your SPA router. Most of our consumers use `react-router-dom` for managing SPA routing, so below is
-an example of how to do this in V5 and V6 of `react-router-dom`.
-[Read more about moving from Router DOM v5 to v6](https://reactrouter.com/en/main/upgrading/v5)
+We've removed `onAction` from `Breadcrumbs.Link`. `Breadcrumbs.Link` now defaults to redirecting
```

Comment:
The [original sentence](https://github.com/Workday/canvas-kit/blame/prerelease/major/modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx#L316) here references removing `onAction` from `Breadcrumbs.Item`, but I believe it's supposed to be `Breadcrumbs.Link`.

Reply:
The `onAction` prop was on `Breadcrumbs.Link` and `Breadcrumbs.Item`. It will be removed from both.